plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -126,6 +127,7 @@
     private static final String FIELDS_NAME_KEY = "name";
     private static final String FIELDS_TYPE_KEY = "type";
     private static final String FIELDS_HIDDEN_KEY = "hidden";
+    private static final String FIELDS_DB_REF_KEY = "dbRef";
```

Comment:
I would recommend storing MongoDB/Bson type name instead of boolean flag.

Reply:
By the way, even if you add a new field here, existing environments won't have the field and it won't be populated forever unless recreating the table. Is this field really needed? Is it covered in the tests?

Reply:
> Is this field really needed?

It can be omitted if MongoSession#execute does the job of finding if the field is DBRef or not. Will this be okay?

Reply:
This field is not required. As we set dbRef flag during `MongoMetadata#applyProjection`. I will remove it.